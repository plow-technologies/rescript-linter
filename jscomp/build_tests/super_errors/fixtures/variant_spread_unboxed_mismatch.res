@unboxed type a = One(int) | Two(string)
type b = | ...a | Three(bool)
