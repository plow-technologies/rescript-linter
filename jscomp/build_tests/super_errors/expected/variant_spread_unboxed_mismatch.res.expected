
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/variant_spread_unboxed_mismatch.res[0m:[2m2:15[0m

  1 [2mâ”‚[0m @unboxed type a = One(int) | Two(string)
  [1;31m2[0m [2mâ”‚[0m type b = | ...[1;31ma[0m | Three(bool)
  3 [2mâ”‚[0m 

  This variant is unboxed, but the variant where this is spread is not. Both variants unboxed configuration must match.